src/chat.cpp

```diff
@@ -43,7 +43,7 @@ ChatBuffer::~ChatBuffer()
 {
 }
 
-void ChatBuffer::addLine(std::wstring name, std::wstring text)
+void ChatBuffer::addLine(const std::string &name, const std::string &text)
```

Comment:
This should stay `wstring`.  If you use an std::string the lines won't wrap properly because this will think that a multibyte sequence is multiple characters and therefore it will cut off the text early and may cut in the middle of MB sequences.